### ISSUE TITLE
doc indicates to import "gopkg.in/alessio/shellescape.v1", but module declares al.essio.dev/pkg/shellescape

### ISSUE THREAD

**Issue by antoinetran** [alessio/shellescape#28]
See https://pkg.go.dev/github.com/alessio/shellescape#section-readme
Using playground from https://play.golang.org/p/HJ_CXgSrmp results in 
```
go: finding module for package github.com/alessio/shellescape
go: downloading github.com/alessio/shellescape v1.5.0
go: found github.com/alessio/shellescape in github.com/alessio/shellescape v1.5.0
go: play imports
	github.com/alessio/shellescape: github.com/alessio/shellescape@v1.5.0: parsing go.mod:
	module declares its path as: al.essio.dev/pkg/shellescape
	        but was required as: github.com/alessio/shellescape
```

Please fix it by matching the doc and the `go.mod`, so if we keep what is in this file, it should be al.essio.dev/pkg/shellescape

**Comment by antoinetran** [comment]
Linked to https://github.com/alessio/shellescape/issues/25

**Comment by antoinetran** [comment]
However I don't know if, for the long term, we should rely on the DNS name `al.essio.dev` (what if the name is not renewed one day ^^), or public name like `github.com` or `go.pkg.in`

**Comment by alessio** [comment]
Thanks for taking the time to bring this problem up!
This is now fixed in aaff7ce29c4ecd9651510be4c29ae73b17497b46, hence closing.
